File: ChatSupport/src/main/java/com/vantagecircle/chatapp/core/interfaceC/FileInterface.java
package com.vantagecircle.chatapp.core.interfaceC;

import com.vantagecircle.chatapp.core.model.FileModel;

/**
 * Created by bapidas on 04/08/17.
 */

public interface FileInterface {
    void onProgress(FileModel fileModel);
    void onPause(FileModel fileModel);
    void onFail(FileModel fileModel);
    void onComplete(FileModel fileModel);
}

File: app/src/main/java/com/bapidas/chattingapp/data/core/callbacks/FileInterface.kt
package com.bapidas.chattingapp.data.core.callbacks

import com.bapidas.chattingapp.data.core.model.FileModel

/**
 * Created by bapidas on 04/08/17.
 */
interface FileInterface {
    fun onProgress(fileModel: FileModel)
    fun onPause(fileModel: FileModel)
    fun onFail(fileModel: FileModel)
    fun onComplete(fileModel: FileModel)
}